### PR TITLE
CHI-1526: Search result styling tweaks

### PR DESCRIPTION
**Pull request by stephenhand**
Primary reviewer: @mmythily 

## Description

* Small tweaks to the layout requested by Jana
* Fix some misaligned translations

### Checklist
- [X] Corresponding issue has been opened
- N/A New tests added
- N/A Feature flags added
- N/A Strings are localized
- [X] Tested for chat contacts
- [X] Tested for call contacts

### Related Issues
Fixes CHI-1580

### Verification steps

* See ticket
* eyeball the case & contact search results
* test the 'see more' and 'see less' function

### CODE REVIEW

File: plugin-hrm-form/src/translations/en-US/flexUI.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "CallSummary-None": "- No call summary -",
   "CallSummary-ViewFull": "See full record",
   "CallSummary-MoreNotes": "more notes",
+  "CallSummary-ContactDate": "Contact Date",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/en-US/flexUI.json:13]: Is this one actually used?

**Review by mythilytm — APPROVED** [review]
Looks great!

File: plugin-hrm-form/src/translations/en-US/flexUI.json (hunk 1 above)
Comment (GPaoloni) [reply plugin-hrm-form/src/translations/en-US/flexUI.json:13]: Looks like it does not (neither does the counselor one).
Screenshot from 2022-12-15 13-22-11

**Review by GPaoloni — APPROVED** [review]
LGTM! Styles are on point. Just a small comment about the strings.

**Pull request merged**